.github/workflows/comment-commands.yaml

```diff
@@ -0,0 +1,56 @@
+name: Comment Commands
```

Comment:
Very nice - thanks for moving all this to its own file - excited to see the emoji and label addition once we merge to main 🚀